modules/Core/queries/debug_link.php

```diff
@@ -179,10 +179,36 @@
     $logs[$type] = file_exists($file_path) ? Util::readFileEnd($file_path, $max_bytes = 10_000) : '';
 }
 
+$user_data = [];
+if ($user->isLoggedIn()) {
+    $user_groups = [];
+    foreach ($user->getAllGroupIds() as $group_id) {
+        $user_groups[] = $group_id;
```

Comment:
Is this any different than doing "$user_groups = $user->getAllGroupIds();"?

Reply:
that returned stupid 8 => 8, 3 => 3 etc

Reply:
array_values($user->getAllGroupIds());

Reply:
That worked :D